lib/ts/recipe/thirdparty/components/themes/signInAndUp/providersForm.tsx

```diff
@@ -64,9 +66,44 @@ export const ThirdPartySignInAndUpProvidersForm: React.FC<SignInAndUpThemeProps>
         }
     };
 
+    const providers = useMemo(() => {
+        const usesDynamicLoginMethods = SuperTokens.usesDynamicLoginMethods === true;
+        if (usesDynamicLoginMethods === false) {
+            return props.providers.map((provider) => ({
+                id: provider.id,
+                buttonComponent: provider.getButton(),
+            }));
+        }
+        const tenantProviders = ThirdParty.tenantProviders || [];
```

Comment:
I think we could get this directly from the multi-tenancy recipe. I'm not sure if that'd be better, but setting this in `supertokens.tsx` feels very out of place.